### PR TITLE
MG-1018 add data tags for image icons

### PR DESCRIPTION
**Pull request by szhu1026**

### CODE REVIEW

File: scripts/htmlToReact.js [hunk 1]
```diff
@@ -31,7 +31,8 @@ const convertToJsx = (html) => {
             }
 
             const attr = obj.svg[attrKey];
-            const openingTagWithClass = `<svg className={ [this.props.className, '${attr.class}'].join(' ')} `;
+            const dataTag = attr.class === undefined ? "" : `${attr.class.replace("dibsvg-svg ", "")}`;
```
Comment (hellatan) [inline scripts/htmlToReact.js]: can you not get the file name within this script instead?

Comment (hellatan) [inline scripts/htmlToReact.js]: so i think it'd be better to modify the signature to this method when it is called from here: https://github.com/1stdibs/dibs-vg/blob/master/scripts/generateDist.js#L52. There's the `name` variable that you can pass along with the `contents`.

I think it'd be better to pass the `name` argument first then the `contents`:

`htmlToReact(contents)` -> `htmlToReact(name, contents)`

and then in this file, you can use that variable directly instead or do any additional manipulation you need (ie. toLowerCase() maybe)

**Comment by szhu1026** [comment]
@hellatan updated.  could you take a look again?

File: scripts/generateDist.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ svgs.forEach(file => {
 
     // generate react module :
     promises.push(new Promise((resolve, reject) => {
-        htmlToReact(contents).then(result => {
+        htmlToReact(contents, name).then(result => {
```
Comment (hellatan) [inline scripts/generateDist.js]: i'd flip flop them: `htmlToReact(name, contents)`

File: scripts/htmlToReact.js [hunk 3]
```diff
@@ -22,7 +22,7 @@ module.exports = Component;
 `;
 };
 
-const convertToJsx = (html) => {
+const convertToJsx = (html, dataTag) => {
```
Comment (hellatan) [inline scripts/htmlToReact.js]: flip flop the signature and then call `dataTag` `fileName` instead: `const convertToJsx = (fileName, html) => { ... }`

**Comment by szhu1026** [comment]
@hellatan made those updates.

**Comment by hellatan** [comment]
@szhu1026 this good to merge now?

**Comment by szhu1026** [comment]
@hellatan yup!

**Pull request merged**